downloadDirectory/src/main/resources/run_protege_exporter.pl

```diff
@@ -0,0 +1,9 @@
+#! /usr/bin/perl
+
+use GKB::WebUtils;
+
+my $wu = GKB::WebUtils->new_from_cgi();
+my $id = @ARGV[0];
+my $file_name = @ARGV[1];
```

Comment:
To access individual element values from the @ARGV array, it needs to be `$ARGV[0]` and `$ARGV[1]`

Reply:
I dunno... `@ARGV[0]` seems to work too. What's the difference, why should I change it?

Reply:
Weird... it only complained about `@ARGV[0]` _after_ I changed to the line from `my $id = @ARGV[0];` to `my $id = @ARGV[0] || die "No ID specified specified.\n";`

Perl is weird. ;)

Reply:
The "$" symbol is used any time you are referring to a single value even it is from an array.  Whereas the "@" symbol is used when you refer to a list of values.  If you were taking an array slice, for example.  You can take a look at the section "Variable Names and Sigils" here if you're interested: http://modernperlbooks.com/books/modern_perl/chapter_03.html

It is allowed to use the "@" symbol, but it isn't good form.  With warnings enabled, you'll get this:
```
Scalar value @ARGV[0] better written as $ARGV[0] at test.pl line 6.
Scalar value @ARGV[1] better written as $ARGV[1] at test.pl line 7.
```